src/components/CoverButtons.jsx

```diff
@@ -0,0 +1,13 @@
+import heart from "../assets/icons/heart.svg";
+import dots from "../assets/icons/dots.svg";
+import play from "../assets/icons/play.svg";
+
+export const CoverButtons = () => {
+  return (
+    <div className="CoverButtonsDiv fade">
+      <img className="CoverButton" src={heart} alt="play" />
```

Comment:
Nice remembering to add the alt text description! Maybe the heart and dots could have different alt text  :)